invenio_app_ils/ui/src/invenio_app_ils/common/config.js

```diff
@@ -5,3 +5,7 @@ const getInvenioConfig = () => {
 };
 
 export const invenioConfig = getInvenioConfig();
+
+// NOTE: the delay that is used when we fire a request with setTimeOut
+// in order to give some time to elastic search to index.
+export const TIMEOUT_DELAY = 3000;
```

Comment:
Thanks for this. Can you please replace all occurrences of `3000` with this new config?

Reply:
@ntarocco I was planning to do it in a clean separate pr.